helpers/lib/common.js

```diff
@@ -0,0 +1,38 @@
+function isObject(val) {
+    return val !== null && typeof val === 'object';
+}
+
+function isOptions(val) {
+    return isObject(val) && val.fn;
+}
+
+function isString(val) {
+    return typeof val === 'string';
+}
+
+function isEmptyObject(val) {
+    return isObject(val) && Object.keys(val).length === 0;
+}
+
+function isTruthy(val) {
+    if (Array.isArray(val)) {
+        return !!val.length;
+    } else if (isEmptyObject(val)) {
+        return false;
+    } else {
+        return !!val;
+    }
+}
+//function to check if two values are equal also does a check for NaN === NaN
+function equals(val1, val2) {
```

Comment:
I think now that we have an equivalence for this, we really don't need this method anymore, let's just use `Object.is` whenever we need that behavior. I don't see a need for it in the current PR.

Reply:
I have removed the function.